pkg/controller/core.oam.dev/v1alpha2/application/assemble/testdata/apprevision.yaml

```diff
@@ -1,17 +1,19 @@
 apiVersion: core.oam.dev/v1beta1
 kind: ApplicationRevision
 metadata:
+  annotations:
+    oam.dev/kubevela-version: undefined
   labels:
-    app.oam.dev/app-revision-hash: f14d04f6c3210336
+    app.oam.dev/app-revision-hash: 883efe1e84fb0964
     app.oam.dev/name: test-assemble
   name: test-assemble-v1
   namespace: default
   ownerReferences:
   - apiVersion: core.oam.dev/v1beta1
-    controller: false
+    controller: true
```

Comment:
why change to true?

Reply:
Before, we add app as appRevision's owner with `controller: false`. I dont' know why but I think it should be `controller:true`. Now appRevision will use app as its ctrl-owner.

Reply:
Consider the case, when application is deleted, will the apprevision be deleted? I don't think app is the owner of revision, once created the revision is independent.

Reply:
I'm sorry, I'm wrong, the apprevision should be deleted after application is deleted. Or we can't delete them and create a different app with same name.